recipes/leftbar/general_row/general_row.vue

```diff
@@ -3,7 +3,7 @@
     :class="leftbarGeneralRowClasses"
     data-qa="dt-leftbar-row"
   >
-    <button
+    <dt-button
```

Comment:
I intentionally didn't use `dt-button` to avoid CSS redundancies or overriding CSS. I inspected the rendered markup and it's adding structures that changes its and could introduce unintended effects.

`d-btn` has sooo many styles, and there's no reason for them to be bundled in.

/c @braddialpad

Reply:
Rollbacked to use the native `button` element https://github.com/dialpad/dialtone-vue/pull/778/commits/e8d00b8a4eaaf001585fe927a12f0f2f39cc2a61